gtdynamics/universal_robot/Joint.cpp

```diff
@@ -357,6 +372,32 @@ gtsam::Vector6_ Joint::twistConstraint(uint64_t t) const {
   return twist_c_hat - twist_c;
 }
 
+/* ************************************************************************* */
+gtsam::Vector6_ Joint::twistAccelConstraint(uint64_t t) const {
```

Comment:
Might be OK if g(x) is the "constraint function", and we can vaguely abuse notation here and call it "Constraint", otherwise too long.

Reply:
Let's call it "Constraint" as it is now.